doc/commands/update.rst

```diff
@@ -12,14 +12,17 @@ nextstrain update
 
 .. code-block:: none
 
-    usage: nextstrain update [-h] [<runtime>]
+    usage: nextstrain update [<pathogen-name>[@<version>] | <runtime-name> […]]
+           nextstrain update
+           nextstrain update --help
 
 
-Updates a Nextstrain runtime to the latest available version, if any.
+Updates Nextstrain pathogens and runtimes to the latest available versions, if any.
 
-The default runtime (docker) is updated when this command is run
-without arguments.  Provide a runtime name as an argument to update a specific
-runtime instead.
+When this command is run without arguments, the default version for each set up
```

Comment:
```suggestion
When this command is run without arguments, the default version for each setup
```

Reply:
I think it's correct as is, technically? The full sentence reads:

> When this command is run without arguments, the default version for each set up pathogen (none) and the default runtime (docker) […]

Reply:
yeah, I think you're right, suggestion withdrawn.